code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -475,9 +477,22 @@
 			adjustBruteLoss(30)
 
 /mob/living/simple_animal/adjustBruteLoss(damage)
-	health = Clamp(health - damage, 0, maxHealth)
-	if(health < 1)
-		Die()
+	if(!ignored_damage_types[BRUTE])
+		health = Clamp(health - damage, 0, maxHealth)
+		if(health < 1)
+			Die()

```

Comment:
Not sure why this acts differently than the tox and cloneloss procs, shouldn't you just be able to check if the mob ignores a damage type and call `..(damage)`?